atsign-foundation/atsign.dev-3.0
# fix: Reset left nav capitalization

**- What I did**
Fixed #156 
**- How I did it**
Added the `text-transform: capitalize` in _dark.scss file
![image](https://user-images.githubusercontent.com/72661784/193589535-ac86ddb9-6005-458d-a59e-fda12fef2dc4.png)

**- How to verify it**
In the home page you can see the nav-bar to verify.
The ss:
![image](https://user-images.githubusercontent.com/72661784/193588950-347ba26f-4695-4733-b795-570b26b59af2.png)

**- Description for the changelog**
Added property of text-transform to change the caps-lock on the tiles.
Note: the change in hugo_stats.json file was not made by me, it got changed when I did `npm install`
